### PR TITLE
Changed Colors of All Sections

### PR DESCRIPTION
**Pull request by yazh1202**
image
Was using Prettier for formatting but for some reason VsCode was adding indent on saving even when I had formatting off on save.

### CODE REVIEW

**Comment by makoteq** [comment]
hi! @yazh1202 
Your changes looks great so far,
But i still have some issues:

File: index.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ ${chalk.yellowBright.bold('Options: ')}
           fileSentences(data);
           // Get and display the absolute path of file.
           console.log(
-            `${chalk.blueBright.bold(
```
Comment (makoteq) [inline index.js:129]: provide matching highlight of text for `stats `
image

**Review by makoteq — CHANGES_REQUESTED** [review]
also change `engine.js` colors to match yellow-white appearance

**Comment by makoteq** [comment]
after that we should be good to go! 😺

File: index.js (hunk 1 above)
Comment (yazh1202) [reply index.js:129]: Screenshot from 2022-10-26 20-54-15
Done

Comment (yazh1202) [reply index.js:129]: Also changed the white color to whiteBright color to match the system default for white text.

**Comment by makoteq** [comment]
Now its perfect,
Good job!

**Pull request merged**